source/appModules/calculator.py

```diff
@@ -6,13 +6,30 @@
 """App module for Windows 10 Calculator.
 This is also the base app module for Windows 11 Calculator."""
 
+# This file has some fairly tricky logic, so here is a test plan to make sure
```

Comment:
You don't have to write a test plan in the app module files - I think we should open a new discussion on adding test plans for Microsoft Store apps such as Calculator.

Reply:
Ok, I deleted the test plan for now.  If the team does eventually decide you'd like to have such a thing, I'm happy to re-add it wherever you think it should go.

Reply:
Please add manual test plans to `tests/manual`. Manual test plans are welcome per the contributing guide https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/githubPullRequestTemplateExplanationAndExamples.md#code-review-checklist